python/python/res/job_queue/ext_job.py

```diff
@@ -137,6 +143,31 @@ def get_max_running_minutes(self):
     def set_max_running_minutes(self, min_value):
         self._set_max_running_minutes(min_value)
 
+    def minimumArgumentCount(self):
+        return self._min_arg()
+
+    def maximumArgumentCount(self):
+        return self._max_arg()
+
+    def argumentTypes(self):
+        """ @rtype: list of type """
+
+        result = []
+        for index in range(self.maximumArgumentCount()):
```

Comment:
`maximumArgumentCount` should be snake_case and also a propery?  Also perhaps it could be `__len__`?

Reply:
Good comments - as mentioned by @stefoss23 these comments will be adressed in a current work. The `maximumArgumentCount` represents a maximum possible number of arguments, and should not be implemented as `__len__`.